org.eclipse.lsp4e/src/org/eclipse/lsp4e/outline/SymbolsModel.java

```diff
@@ -74,6 +68,29 @@ public int hashCode() {
 		}
 	}
 
+	public static class DocumentSymbolWithURI extends DocumentSymbolWithFile {
```

Comment:
Would it not be easier to extend DocumentSymbol instead of doing composition?

Reply:
I understand, shall we merge then now?

Reply:
Yes please.

Reply:
thanks